roles/core/nic/readme.rst

```diff
@@ -51,7 +54,11 @@ To configure an LACP bonding, specify slave interfaces, and then create the bond
       type: bond-slave
       master: bond0
 
-To configure a vlan, simply set vlan to true:
+To configure a VLAN, simply set vlan to true.
+
+In this example we want to use a dedicated subnet for VLAN 100, we have created a net-100 network (corresponding to VLAN 100).
```

Comment:
Can you split line in 2 ? So users can easily read in a shell.

Reply:
Done